styles/SASS/app.scss

```diff
@@ -0,0 +1,33 @@
+@import './resets';
+@import './loading-view';
+@import './start-view';
+@import './pokemon-choose-view';
+@import './fight-view';
+@import './animations';
+@import './config';
```

Comment:
The order of imports matters.
Import of the config.scss file should be at the beginning (right after the style reset file) - because other files such as start-view.scss can also use the variables from config file.

Reply:
Good point. I changed it.